linux/dm-sworndisk/src/bio.rs

```diff
@@ -0,0 +1,350 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
Why this PR described with "crypto" contains block device related code? Can they be divided into independent PR?